markdown/blog/2024-marketing-report.md

```diff
@@ -0,0 +1,177 @@
+
+- Implemented regular content updates on our platforms to keep followers informed and engaged
+
+- Proposed the Student Ambassador Program initiative that aims to bring more attention to AsyncAPI's mission and tools, further building our presence and strengthening community ties
+
+## Implementing Buffer for Social Media Management
+
+To improve the efficiency and security of our social media management, we’ve introduced a paid version of Buffer for posting content across our platforms. This change enables us to streamline content scheduling, ensuring consistent, professional updates without compromising security. Importantly, using Buffer restricts access to the main social media accounts, adding a layer of protection and allowing us to manage permissions efficiently.
+
+Here are some of the benefits of using the Buffer subscription
+- **Centralized Dashboard:** Manage all social media platforms from a single interface, making tracking and organizing content easier. 
+- **Improved Security:** No need for multiple people to log into the actual social media accounts. This minimizes the risks of unauthorized changes or breaches.
+- **Better Analytics and Insights:** Access in-depth analytics to understand which posts perform and engagement metrics best and adjust strategies accordingly. 
+- **Content Approval Workflow:** Allow team members to draft posts for review, ensuring high-quality content that aligns with AsyncAPI’s voice.
+
+
+Please note that I currently do not have direct access to AsyncAPI’s social media accounts. Buffer provides an effective solution for posting content, monitoring performance, and maintaining the high standards of AsyncAPI's public presence without broad access.
+
+As part of the benefit of using Buffer, we have access to our social media analytics. Below are the analytics for LinkedIn from January 1st to December 12th, 2024.
+
+
+
+<Figure
+  src="/img/posts/marketing-images/linkedIn-metrics-insights.webp"
+  alt="Graph showing AsyncAPI LinkedIn follower growth"
+  caption="AsyncAPI LinkedIn graph showing the followers growth from Buffer analytics."
+  className="text-center"
+/>
+
+
+
+<Figure
+  src="/img/posts/marketing-images/linkedin-performance-statistics.webp"
+  alt="Image showing AsyncAPI LinkedIn performance including follower growth, engagement rates, and post performance over time"
+  caption="AsyncAPI LinkedIn performance statistics from Buffer analytics."
+  className="text-center"
+/>
+ 
+
+
+
+In 2024, our LinkedIn performance saw impressive growth: a 48% increase in followers (3,910 total), 104% more impressions (100,328), and 91% more new followers (911). Engagement soared with a 219% rise in clicks (4,295 total) and a 9% improved engagement rate (5.32%). We also published 44% more posts (233), highlighting consistent content efforts. It has been difficult to track the exact numbers on X formally (Twitter). Hence, we will not be sharing any data from X at this time.
```

Comment:
```suggestion
In 2024, our LinkedIn performance saw impressive growth: a 48% increase in followers (3,910 total), 104% more impressions (100,328), and 91% more new followers (911). Engagement soared with a 219% rise in clicks (4,295 total) and a 9% improved engagement rate (5.32%). We also published 44% more posts (233), highlighting consistent content efforts. 

Tracking our performance on X (formally Twitter) has not been easy. Buffer does not have access to historical data. We can see numbers that show our performance since we connected Buffer in September. We do not have data that we could share and show how our performance improved as we do not have such.
```